binary/BUILD

```diff
@@ -15,5 +15,6 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
+#
 
-exports_files(["release_validate_deps.py"])
+exports_files(["grakn", "grakn.bat", "grakn-docker.sh"])
```

Comment:
new line

Reply:
Solved in https://github.com/graknlabs/dependencies/pull/136/commits/e44a270ac97b4d28e56e409d91fd75663ccb164c